docs/source-2.0/guides/generating-code/index.rst

```diff
@@ -0,0 +1,21 @@
+.. _generating-code:
+
+===============
+Generating code
+===============
+
+One of the greatest strengths of Smithy is code generation. Smithy models can be used
+to generate clients and servers in a variety of programming languages. The generated
+code takes care of low level details like endpoints, serialization/deserialization, etc.
```

Comment:
**Minor**: I think changing the end of this phrase to something like "[...] endpoints, serialization, deserialization and others." sound a little bit better (: